src/engine/bpmcontrol.cpp

```diff
@@ -834,9 +841,9 @@ void BpmControl::setInstantaneousBpm(double instantaneousBpm) {
 
 void BpmControl::resetSyncAdjustment() {
     // Immediately edit the beat distance to reflect the new reality.
-    double new_distance = m_pThisBeatDistance->get() + m_dUserOffset;
+    double new_distance = m_pThisBeatDistance->get() + m_dUserOffset.getValue();
```

Comment:
Why getValue/setValue instead of fetchAndStore?

Reply:
It is ControlValueAtomic, which has no fetchAndStore().